HTML/CSS & JavaScript capstone project - Conference page

SlowKingV opened the pull request:
# Description
In this capstone project, you are going to build is based on an online website for a conference. We provide some design guidelines for you to create the website, but you must personalize the content, i.e., instead of a conference, you can build a website for a concert or for a web development course. It's very important that you personalize your project so that you have something unique in your portfolio to share with potential employers during job searching.

## Project Requirements
- You should personalize the content of your page. Choose a topic that is different than the one in the original design.
- You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
  - Typography: font face, size and weight.
  - Layout: composition and space between elements.
- The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
- You can use a CSS framework (for example Bootstrap) for styling, if you want to.
- You should build only these 2 pages:
  - The home page.
  - The about page.
- Each of these pages should have versions for 2 different screen sizes:
  - Mobile: up to 768px wide.
  - Desktop: 768px or wider.
- Interactions
  - Links
    - The home page should have a link in the menu to the about page.
    - The logo in the header links to the home page.
  - Mobile menu
    - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
    - There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
    - The mobile menu should have a close (X) button that closes the menu.
- Dynamic page
  - The section "Featured speakers" should be created dynamically in JavaScript.
  - You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

## Project Documentation
For this project, we are also required to record a video presenting our project. Here is the link to the video I recorded: https://www.loom.com/share/9b5dd01eba6341259a7b719b66aa6530

zainsadaqat commented on index.html:
```diff
@@ -0,0 +1,117 @@
+    </section>
+    <section id="program">
+      <h2 class="program-title">Main Program</h2>
+      <ul class="program-events">
+        <li class="event">
+          <img src="./assets/program_icon_01.png" alt="" class="icon event-info">
+          <h3 class="name event-info">Conference</h3>
+          <p class="description event-info">Listen to music producers and aritst talk about their way to the top.</p>
+        </li>
+        <li class="event">
+          <img src="./assets/program_icon_02.png" alt="" class="icon event-info">
+          <h3 class="name event-info">Exhibiton</h3>
+          <p class="description event-info">Appreciate the different instruments and merchandising from your favorite idols.</p>
+        </li>
+        <li class="event">
+          <img src="./assets/program_icon_03.png" alt="" class="icon event-info">
+          <h3 class="name event-info">Forum</h3>
+          <p class="description event-info">Have the time to share your thoughts and opinions with all the community online.</p>
+        </li>
+        <li class="event">
+          <img src="./assets/program_icon_04.png" alt="" class="icon event-info">
+          <h3 class="name event-info">Workshop</h3>
+          <p class="description event-info">Try to produce your own music using the different sample some of our featured producers made specially for the event.</p>
+        </li>
+        <li class="event">
+          <img src="./assets/program_icon_05.png" alt="" class="icon event-info">
+          <h3 class="name event-info">Retro Ignite</h3>
+          <p class="description event-info">Get opportunities to network with RE affiliates around the world, also after the festival.</p>
+        </li>
+      </ul>
```
The "Main Program" section doesn't look consistent. As you can see in the screenshots, the description isn't looking consistent in all cards. Kindly work on it so that it'll look consistent. 👍🏻 👍🏻 

[Desktop Version]
<img width="1119" alt="Screenshot 2022-07-02 at 10 24 55 PM" src="https://user-images.githubusercontent.com/83048208/177010332-71308add-14cf-4077-9b1d-0d99f72d3106.png">

[Mobile Version]
<img width="379" alt="Screenshot 2022-07-02 at 10 25 19 PM" src="https://user-images.githubusercontent.com/83048208/177010349-3541d8d9-35b3-4279-9554-b5914ab71547.png">

zainsadaqat reviewed (changes requested):
# Required Changes 🔁
# Hello @SlowKingV  😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [x] As you can see in the screenshot, in the speakers section, all speakers images have a checkbox image right behind them. Kindly make it look like that.
[YOUR DESIGN]
<img width="560" alt="Screenshot 2022-07-02 at 10 27 54 PM" src="https://user-images.githubusercontent.com/83048208/177010426-2339c39e-0da3-450f-9fb5-f0f3adc19f18.png">
[REQUIRED DESIGN]
<img width="394" alt="Screenshot 2022-07-02 at 10 29 04 PM" src="https://user-images.githubusercontent.com/83048208/177010474-a031e278-ca8c-4dae-b8c2-bedf72094adc.png">

_Check the comments under the review._


<img src="https://media0.giphy.com/media/QvMIzHw30MmMAqvOeo/giphy.gif"/>

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

SlowKingV commented:
- I have corrected the Featured artist design and added the checkerboard behind as requested.
- I've also improved the consistency between the program boxes.

AmaduKamara reviewed (approved):
# Status: Approved 🟢🟢🟢 

Hi @SlowKingV,

You're doing great, good job working on the changes asked by the previous reviewer. 

![YouNailedItManJackHarlowGIF (2)](https://user-images.githubusercontent.com/50941074/177013162-2c2bc803-7e0d-438a-bbbc-bb0f9c52b52c.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights

✔️ Correct use of Github workflow
✔️ No linter error
✔️ Good commit messages
✔️ README is okay
✔️ Your design is soft and clean 👍 

I'm proud of your design skills, keep it up 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```
